horton/gbasis/cext.pyx

```diff
@@ -724,6 +724,50 @@ cdef class GOBasis(GBasis):
         # done
         return output
 
+    def compute_multipole_moment(self,
+                    np.ndarray[long, ndim=1] xyz not None,
+                    np.ndarray[double, ndim=1] center not None,
+                    output):
+        """Compute the (multipole) moment integrals in a Gaussian orbital basis
+           < gto_a | (x - C_x)^l (y - C_y)^m (z - C_z)^n | gto_b >
+
+           **Arguments:**
+
+           xyz
+                A integer (long) numpy-array with shape (3) with the powers of x,y,z in the integrals,
+                shape = (3,).
+
+           center
+                A numpy array with shape (3) with the center [C_x, C_y, C_z] around which
+                the moment integral is computed,
+                shape = (3,).
+
+           output
+                When a ``TwoIndex`` instance is given, it is used as output
+                argument and its contents are overwritten. When ``LinalgFactory``
+                is given, it is used to construct the output ``TwoIndex``
+                object. In both cases, the output two-index object is returned.
+
+           **Returns:** ``TwoIndex`` object
+        """
```

Comment:
do you mean changing it to something like this?

        """Compute the (multipole) moment integrals in a Gaussian orbital basis.

        < gto_a | (x - C_x)^l (y - C_y)^m (z - C_z)^n | gto_b >

        Parameters
        ----------

        xyz
            A integer (long) numpy-array with shape (3) with the powers of x,y,z in the integrals,
            shape = (3,).

        center
            A numpy array with shape (3) with the center [C_x, C_y, C_z] around which
            the moment integral is computed,
            shape = (3,).

        output
            When a ``TwoIndex`` instance is given, it is used as output
            argument and its contents are overwritten. When ``LinalgFactory``
            is given, it is used to construct the output ``TwoIndex``
            object. In both cases, the output two-index object is returned.

        Returns
        -------
        ``TwoIndex`` object

        """